### PR TITLE
Verify docker image can be built

### PR DESCRIPTION
**Pull request by mst-ableton**
This closes https://github.com/AbletonAppDev/devtools/issues/1458.

@ala-ableton @rco-ableton @nre-ableton @eta-ableton ptal!

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
+sudo: false
+
 language: python
+
 python:
     - "3.6"
-# command to install dependencies
+
+# Install dependencies.
 install: "pip install -r requirements.txt flake8 flake8-commas yamllint"
```
Comment (ala-ableton) [inline .travis.yml]: You might consider using the same style as for `script`:
```yaml
install:
    - pip install -r requirements.txt flake8 flake8-commas yamllint
```

Comment (mst-ableton) [reply .travis.yml]: This is done in 573b175

**Comment by eta-ableton** [comment]
VNLGTM 👍

File: .travis.yml (hunk 1 above)
Comment (ala-ableton) [reply .travis.yml]: I know. I was reviewing the commits in order and thought that it would be nice to do in the first one.

**Pull request merged**